string operation
string hand

---

bool succes